inductiva/fluids/wind_tunnel/README.md

```diff
@@ -22,22 +22,29 @@ Launching a simulation returns a task object, which can be used to verify the st
 
 ### Example:
 
-To run this example download a vehicle geometry in OBJ format from [here](/resources/geometry/test_vehicle.obj) and
-do not forget to insert your API Key (get one by filling this [form](https://docs.google.com/forms/d/e/1FAIpQLSflytIIwzaBE_ZzoRloVm3uTo1OQCH6Cqhw3bhFVnC61s7Wmw/viewform?usp=sf_link)).
+To run this example you need an object. You can fetch any object you like directly with Inductiva package. On this example, we download a test object saved in our Github repository.
+
+Do not forget to insert your API Key (get one by filling this [form](https://docs.google.com/forms/d/e/1FAIpQLSflytIIwzaBE_ZzoRloVm3uTo1OQCH6Cqhw3bhFVnC61s7Wmw/viewform?usp=sf_link)).
 
 ```python
 import inductiva
 
 inductiva.api_key = "YOUR_API_KEY"
 
+# Url to a test object in Inductiva Github repository
+object_url = "https://raw.githubusercontent.com/inductiva/inductiva/main" \
+              "/resources/test_vehicle.obj"
+test_object_path = inductiva.utils.files.download_from_url(
```

Comment:
Still thinking if it makes sense to have the method under utils.files, or just utils... will have a look...

Reply:
It's a little verbose, but I think it's fine for now. I saw that there are several other util modules under the "utils" folder, and having this method on the files.py makes sense.